src/containers/project-container/project-container.test.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
use waitFor to delay execution of test assertions until certain stuff happens. Be sure to mark the test case method as async. `await waitFor(() => screen.getByText('1.1.0 - Impact Attenuator'))`